twingate/client_connector.go

```diff
@@ -20,13 +20,20 @@ type Connectors struct {
 
 const connectorResourceName = "connector"
 
-type createConnectorQuery struct {
+type createConnectorMutation struct {
```

Comment:
Please roll back this change from query to mutation even though it is correct this is currently not our standard and we need to change it everywhere if we do it, so this is not the appropriate PR for this change.